### ISSUE TITLE
Compose V3 example

### ISSUE THREAD

**Issue by Vad1mo** [fatk/docker-letsencrypt-nginx-proxy-companion-examples#17]
Currently there is no compose v3 example available. 

V3 compose does not support `volumes-from` anymore, so one has to use named volume container.

**Comment by fatk** [comment]
I don't think it will be possible until docker-letsencrypt-nginx-proxy-companion stops [relying on it to find the container](https://github.com/JrCs/docker-letsencrypt-nginx-proxy-companion/blob/1ea4335b510dda7b293865bfc05323f083af64bc/app/entrypoint.sh#L27) 

Maybe you could PR them?
